test/common/network/connection_impl_test.cc

```diff
@@ -1789,6 +1789,30 @@ TEST_F(MockTransportConnectionImplTest, WriteReadyOnConnected) {
       .WillOnce(Return(IoResult{PostIoAction::KeepOpen, 0, true}));
 }
 
+// Test the interface used by external consumers.
```

Comment:
Per @ggreenway I agree it might be a good idea to add something like "As of 2/20, used by Google" or something like that?

Reply:
commented in .h file.